### DISCUSSION TITLE
Welcome to Teams Toolkit Discussions!

### DISCUSSION THREAD

**Discussion by MuyangAmigo** [OfficeDev/microsoft-365-agents-toolkit#8976] [Announcements]
## 👏  Welcome

We're using Discussions as a place to connect with other members of our community. We hope that you:
* Ask questions you're wondering about.
* Share ideas.
* Engage with other community members.
* Welcome others and are open-minded. Remember that this is a community we build together 💪 .

My name is Junjie Li, a Product Manager at Microsoft Developer Division. I'm here to help where I can, listen to your needs and learn from our community 🥳 . 

If you haven't already, visit [Microsoft Teams developer documentation](https://aka.ms/teamsfx-docs) to get started with building apps with Teams Toolkit today 🎉 .

**Comment by cardinalpipkin** [comment]
Hi - thanks for opening this discussion panel. Any chance of some answers to the most of the questions please?
Best regards. And thanks again.
